packages/url_launcher/url_launcher/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,21 +7,25 @@
   -->
   <uses-permission android:name="android.permission.INTERNET"/>
 
+  <!--#docregion android-queries-->
   <!-- Provide required visibility configuration for API level 30 and above -->
   <queries>
     <intent>
```

Comment:
It's probably worth adding a comment saying this is only here for integration tests, and shouldn't be needed in most actual apps, so it doesn't confuse anyone else in the future.

Reply:
Agreed; I've added a comment to the `https` scheme bit of the XML, in addition to excluding it from the relevant docregion.